### PR TITLE
front: Update translations and display errors count based on filter l…

### LINKED ISSUE #6807: Wrong label for objects in the error list view
### What happened?

When opening the error list modal, it says 
image
5890 errors found, even though I asked for warnings ! 
When I select "errors", it says 0 errors found. When selecting "all", it says 5890 errors found.

It just says error for everything

### What did you expect to happen?

I'd expect at least that when selecting "Warnings", it shows "5890 warnings ~problems~".

Ideally too, when selecting "all" it would say "0 errors and 5890 warnings" or if easier "5890 items"

### How can we reproduce it (as minimally and precisely as possible)?

1. Go to the France infrastructure
2. Toggle the infra error layer 
3. Click on the errors count to display the list
4. Move the selector from "All" to "Warnings"

### What operating system, browser and environment are you using?

- **Browser**: Firefox
- **OS**: Windows 11
- **Env**: dev

### OSRD version (top right corner `Account` button > `Informations`)

dev b1f8892

### PR DESCRIPTION
**Pull request by achrafmohye**
Closes #6807

### CODE REVIEW

File: front/public/locales/en/translation.json [hunk 1]
```diff
@@ -159,7 +159,9 @@
         "error": "An error occurred while loading the data",
         "filter-level": "Error level",
         "filter-type": "Error type",
-        "total-error": "{{count}} errors found",
+        "total-errors": "{{count}} errors found",
+        "total-all": "{{count}} errors and warnings found",
+        "total-warnings": "{{count}} warnings found",
         "total-error_one": "One error found",
         "total-error_zero": "No error found",
```
Comment (clarani) [inline front/public/locales/en/translation.json]: You need to add keys _one and _zero for the 2 new keys. You also have to update the existing keys
```suggestion
        "total-errors": "{{count}} errors found",
        "total-all": "{{count}} errors and warnings found",
        "total-warnings": "{{count}} warnings found",
        "total-errors_one": "One error found",
        "total-errors_zero": "No error found",
```

**Review by clarani — CHANGES_REQUESTED** [review]

**Review by clarani — APPROVED** [review]
LGTM, tested :white_check_mark:

File: front/public/locales/fr/translation.json [hunk 2]
```diff
@@ -159,9 +159,15 @@
...
-        "total-error_zero": "Aucune anomalie trouvée",
+        "total-errors": "{{count}} erreurs trouvées",
+        "total-all": "{{count}} erreurs et avertissements trouvés",
+        "total-all_one": "Une erreur ou un avertissement trouvé",
+        "total-all_zero": "Aucune erreur ni avertissement trouvé",
+        "total-warnings": "{{count}} avertissements trouvés",
+        "total-warnings_zero": "Aucun avertissement trouvé",
+        "total-warnings_one": "Un avertissement trouvés",
+        "total-errors_one": "Une erreur trouvée",
+        "total-errors_zero": "Aucune erreur trouvée",
```
Comment (kmer2016) [inline front/public/locales/fr/translation.json]:
```suggestion
        "total-all": "{{count}} erreurs et avertissements trouvés",
        "total-all_one": "Une erreur ou un avertissement trouvé",
        "total-all_zero": "Aucune erreur ni avertissement trouvé",
        "total-warnings": "{{count}} avertissements trouvés",
        "total-warnings_zero": "Aucun avertissement trouvé",
        "total-warnings_one": "Un avertissement trouvés",
        "total-errors": "{{count}} erreurs trouvées",
        "total-errors_one": "Une erreur trouvée",
        "total-errors_zero": "Aucune erreur trouvée",
```

Just group by message type

File: front/public/locales/en/translation.json [hunk 3]
```diff
@@ -159,9 +159,15 @@
         "error": "An error occurred while loading the data",
         "filter-level": "Error level",
         "filter-type": "Error type",
-        "total-error": "{{count}} errors found",
-        "total-error_one": "One error found",
-        "total-error_zero": "No error found",
+        "total-errors": "{{count}} errors found",
```
Comment (kmer2016) [inline front/public/locales/en/translation.json]: same comment than the previous one

**Review by kmer2016 — APPROVED** [review]
LGTM !

**Pull request merged**